docs/examples/ex02.py

```diff
@@ -1,3 +1,4 @@
+from pathlib import Path
```

Comment:
Same comment regarding `literalinclude` in ex02.rst as above.

Reply:
What about replacing `:lines:` for [`literalinclude`](http://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-literalinclude) with `:start-at:` and `:end-at:`?   For example:
```python
.. literalinclude:: ex02.py
    :start-at: skfem
    :end-at: refine
```

Otherwise I think every `:lines:` field downpage of an edit has to be amended.  These are also more readable in the .rst source files.

Reply:
That is a good idea. I didn't know about this option.